.eslintrc

```diff
@@ -1,10 +1,20 @@
 {
-  "plugins": ["prettier"],
-  "extends": ["plugin:prettier/recommended", "eslint-config-atomic"],
+  "root": true,
+  "plugins": [
+    "@typescript-eslint",
+    "only-warn"
+  ],
```

Comment:
As I mentioned before, I am not interested in replacing eslint-config-atomic with typescript-eslint as eslint-config-atomic provides the same features and more rules and does not require any configuration.